src/main/package/www/languages/mc_locale_gui_source_en.json

```diff
@@ -21,7 +21,10 @@
   "CHART.DOWNLOAD_CSV_FILE" : "Download CSV File",
   "CHART.GRAPHICAL_REPORT" : "Graphical Report",
   "CHART.LAST_SEEN" : "Last seen:",
+  "CHART.NODE" : "Node:",
```

Comment:
Can you change as follows the variable types? So that we can use it across all pages like, alarm, timer, etc.,
```
"SUB_HEADER.NODE" : "Node",
"SUB_HEADER.SENSOR" : "Sensor",
"SUB_HEADER.TYPE" : "Type",
"SUB_HEADER.LAST_SEEN" : "Last Seen",
"SUB_HEADER.VARIABLE_TYPE" : "Variable Type",
"SELECT.VARIABLE_TYPE"  : "Select Variable Type",
```